test/start.sh

```diff
@@ -4,19 +4,20 @@ args=
 
 function run {
   file=$1;
+  if [ "${args[0]}" = "${file}" ]; then
+    args=("${args[@]:1}");
+  fi
   echo "";
-  echo "> DEBUG=leoric mocha --exit --timeout 5000 ${file} ${args}";
-  DEBUG=leoric mocha --exit --timeout 5000 ${file} ${args} || exit $?;
+  printf '"%s" ' "${args[@]}" | xargs echo "> DEBUG=leoric mocha --exit --timeout 5000 ${file}";
+  printf '"%s" ' "${args[@]}" | DEBUG=leoric xargs mocha --exit --timeout 5000 ${file} || exit $?;
 }
 
 ##
 # Run unit tests first in order catch bugs as soon as possible
 function unit {
   # recursive glob nor available in bash 3
   # - https://unix.stackexchange.com/questions/49913/recursive-glob
-  for file in $(ls test/unit/{,drivers/,drivers/*/,adapters/}*.test.js); do
-    run ${file};
-  done
+  run "$(ls test/unit/{,drivers/,drivers/*/,adapters/}*.test.js)";
```

Comment:
merged unit tests in the same mocha process, makes following shortcut available:

```bash
$ npm run test:unit -- --grep bulkCreate
```